src/services/service.ts

```diff
@@ -0,0 +1,167 @@
+import { QueryResult } from 'pg'
+const client = require('../postgres/conn')
+client.connect()
+
+//GET
+
+const getUsers = async (_req: Express.Request, res: Express.Response) => {
+    try {
+        await client.query('SELECT * from users', (error: Error, result: QueryResult) => {

```

Comment:
Isn't necessary the callback because you are using try catch and async await

Reply:
done